src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/build/netstandard1.6/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ok. We can probably update https://github.com/aspnet/Configuration/blob/dev/src/Microsoft.Extensions.Configuration.UserSecrets/build/netstandard1.0/Microsoft.Extensions.Configuration.UserSecrets.targets then